tests/modules/programs/tmux/emacs-with-plugins.conf

```diff
@@ -0,0 +1,52 @@
+# ============================================= #
+# Start with defaults from the Sensible plugin  #
+# --------------------------------------------- #
+run-shell @sensible_rtp@
+# ============================================= #
+
+set  -g default-terminal "screen"
+set  -g base-index      0
+setw -g pane-base-index 0
+
+new-session
+
+bind v split-window -h
+bind s split-window -v
+
+
+set -g status-keys emacs
+set -g mode-keys   emacs
+
+
+
+
+
+setw -g aggressive-resize on
+setw -g clock-mode-style  24
+set  -s escape-time       500
+set  -g history-limit     2000
+
+
+
+# ============================================= #
+# Load plugins with Home Manager                #
+# --------------------------------------------- #
+
+# tmuxplugin-logging
+# ---------------------
+
+run-shell /nix/store/ppyx7z9m8n9dkq3p977r8f0sv20rynaf-tmuxplugin-logging/share/tmux-plugins/logging/logging.tmux
+
+
+# tmuxplugin-prefix-highlight
+# ---------------------
+
+run-shell /nix/store/3i3hb0sxn11cf47rbnq4vcil2bpsjiiy-tmuxplugin-prefix-highlight/share/tmux-plugins/prefix-highlight/prefix_highlight.tmux
+
+
+# tmuxplugin-fzf-tmux-url
+# ---------------------
+
+run-shell /nix/store/hhkdfsqdbdd2kagzp1yfbjna8qh77gws-tmuxplugin-fzf-tmux-url/share/tmux-plugins/fzf-tmux-url/fzf-url.tmux
```

Comment:
I will fix the above (currently hardcoded) paths once I get the tests passing on Travis.

Reply:
It would be nice to have some generic way to handle matching files that contain store paths but I have not yet thought of an elegant way to accomplish this.

Reply:
We can still @grahamc 's [mutate](https://github.com/grahamc/nixos-config/blob/master/packages/mutate/default.nix). If `nmt` calls mutate on every file before the `assertFileContent` is called, the we can simply refer to pkgs using `@name@` pattern.

Reply:
Yeah, something like that should work. I'll see if I can sort it out. Maybe for the weekend.